src/test/java/upbrella/be/login/controller/LoginControllerTest.java

```diff
@@ -44,12 +49,21 @@ protected Object initController() {
     @DisplayName("사용자는 카카오 소셜 로그인을 할 수 있다.")
     void kakoLoginTest() throws Exception {
         // given
-        String code = "{\"code\":\"1kdfjq0243f\"}";
+        LoginCodeRequest request = LoginCodeRequest.builder()
+                .code("1kdfjq0243f")
+                .build();
+
+        given(oauthLoginService.getOauthToken(request.getCode(), kakaoOauthInfo))
+                .willReturn(new OauthToken("accessToken", "refreshToken", "tokenType", 3600L));
+        given(kakaoOauthInfo.getLoginUri())
+                .willReturn("loginUri");
+        given(oauthLoginService.processKakaoLogin(anyString(), anyString()))
+                .willReturn(new Properties("카카오 사용자", "010-0000-0000"));
```

Comment:
anyString()이 아니라 구체적으로 명시할 수 있지않나요?

Reply:
처음에 객체로 매핑을 해주니, eq 에러가 발생해서 anyString으로 했는데, 단순 String 값을 명시해주는게 더 좋은것 같습니다

Reply:
참조 주소 차이때문이라면 아래처럼 동등성 비교를 할 수 있습니다 
```suggestion
        given(oauthLoginService.processKakaoLogin(refEq(str1), refEq(str2)))
                .willReturn(new Properties("카카오 사용자", "010-0000-0000"));
```